com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py

```diff
@@ -0,0 +1,2 @@
+class ViewNotFoundError(Exception):
+    """Raised when the SWS view element cannot be located."""
```

Comment:
If we create streamsx.rest.errors.ViewNotFoundError as:
```Python
class ViewNotFoundError(Exception):
    """Raised when the view element resource cannot be located."""
```

Removing "SWS" and adding "resource" to conform to the terminology at [the top of this page](https://www.ibm.com/support/knowledgecenter/en/SSCRJU_4.0.0/com.ibm.streams.restapi.doc/doc/restapis-resource-element.html)